DESCRIPTION

```diff
@@ -27,6 +27,7 @@ Imports:
   methods,
   rmarkdown (>= 2.4),
   rmdfiltr (>= 0.1.3),
+  stringr,
```

Comment:
Interesting, I had not seen that conversation. Our dependency graph has three packages that import `stringr`: `knitr` (where attempts are being made to make it a soft dependency or remove it), `rmarkdown` (where, according to my skimming, it could be easily removed), and `broom` (where it's not essential but would take some work to remove). I'll gladly remove the import, but we should consider starting a conversation with the `broom` developers about this as well if we want to avoid the dependency.